src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -1167,8 +1167,20 @@ skip:
                 + " blocks/sec), " + std::to_string(m_block_queue.get_data_size() / 1048576.f) + " MB queued";
             if (ELPP->vRegistry()->allowed(el::Level::Debug, "sync-info"))
               timing_message += std::string(": ") + m_block_queue.get_overview();
-            MGINFO_YELLOW(context << " Synced " << m_core.get_current_blockchain_height() << "/" << m_core.get_target_blockchain_height()
+            if(m_core.get_target_blockchain_height() == 0){
+              MGINFO_YELLOW(context << " Synced " << m_core.get_current_blockchain_height() << "/" << m_core.get_target_blockchain_height()
                 << timing_message);
+            } else {
+              const int completition_percent = (m_core.get_current_blockchain_height() * 100 / m_core.get_target_blockchain_height());
+              if(completition_percent < 99) {//printing completion percent only if % is < of 99 cause for 99 >= this is useless
+                MGINFO_YELLOW(context << " Synced " << m_core.get_current_blockchain_height() << "/" << m_core.get_target_blockchain_height()
+                  << " " << completition_percent << "% (" << (m_core.get_target_blockchain_height() - m_core.get_current_blockchain_height())
```

Comment:
As per my comment https://github.com/monero-project/monero/pull/4020#issuecomment-398119732, it would be neater to have the percentage inside the parenthesis.

Reply:
I tend to agree. @Jorropo is this something you would be ok with changing?

Alternatively, we could merge this anyway and @jtgrassie can author the change if he desires.